playerbot/strategy/values/AttackersValue.cpp

```diff
@@ -66,34 +67,23 @@ void AttackersValue::AddAttackersOf(Player* player, set<Unit*>& targets)
         return;
 
 	list<Unit*> units;
-	//MaNGOS::AnyUnfriendlyUnitInObjectRangeCheck u_check(player, sPlayerbotAIConfig.sightDistance);
-    //MaNGOS::UnitListSearcher<MaNGOS::AnyUnfriendlyUnitInObjectRangeCheck> searcher(units, u_check);
-    //Cell::VisitAllObjects(player, searcher, sPlayerbotAIConfig.sightDistance);
-
-    list<ObjectGuid> unitGuids = AI_VALUE(list<ObjectGuid>, "possible targets");
-
-    for (auto unitGuid : unitGuids)
-    {
-        Unit* unit = ai->GetUnit(unitGuid);
-        if (unit)
-            units.push_back(unit);
-    }

```

Comment:
This was quite wrong as possible targets was getting the targets from the player owner of this value not the player specified in the argument